js/scripts.js

```diff
@@ -0,0 +1,15 @@
+function getTriangleArea(a, h) {
+	if (a > 0 && h > 0) {
+		return a*h/2;
+	} else {
+		console.log('Nieprawidłowe dane');
```

Comment:
skoro przy dobrym wyniku zwracasz rezultat, to przy złym wyniku też tylko zwracaj rezultat (czyli string z treścią błędu) - dzięki temu funkcja robi jedną rzecz - zwraca rezultat, a za jego publikację (console.log) odpowiada inna część kodu